tests/rptest/services/redpanda.py

```diff
@@ -276,13 +286,25 @@ def __init__(self,
                  cloud_storage_readreplica_manifest_sync_timeout_ms: Optional[
                      int] = None,
                  bypass_bucket_creation: bool = False):
+        self.cloud_storage_type = test_context.injected_args.get(
+            'cloud_storage_type', CloudStorageType.S3)
+
+        if self.cloud_storage_type == CloudStorageType.S3:
+            self.cloud_storage_access_key = cloud_storage_access_key
+            self.cloud_storage_secret_key = cloud_storage_secret_key
+            self.cloud_storage_region = cloud_storage_region
+            self._cloud_storage_bucket = f'panda-bucket-{uuid.uuid1()}'
+
+            self.cloud_storage_api_endpoint = cloud_storage_api_endpoint
+            self.cloud_storage_api_endpoint_port = cloud_storage_api_endpoint_port
+        elif self.cloud_storage_type == CloudStorageType.ABS:
+            self.cloud_storage_azure_shared_key = 'Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=='
+            self.cloud_storage_azure_storage_account = 'devstoreaccount1'
```

Comment:
Note for reviewer: By default, Azurite uses `devstoreaccount1` with the specified shared key. Both are readily available in the docs.